### DISCUSSION TITLE
C8D: What is it?

### DISCUSSION THREAD

**Discussion by spkane** [moby/moby#46569] [Q&A] [answered]
I noticed the term c8d when looking at my Docker setup recently and was trying to figure out what component that relates to. After some investigation, it appears as if this might be related to beta containerd image store support in dockerd (at least in Docker Desktop).

Would be interested in the official word on this and if there is anything that provides an overview and introduction.

Thanks!

Sean

**Comment by rumpl** [comment, marked as answer]
c8d means containerd, we (at least I) are too lazy to write it out all the time. 

It mostly relates to our work around integrating containerd as the image store for images, you can read more about it: https://www.docker.com/blog/extending-docker-integration-with-containerd/

I am curious though, where exactly did you see this?

**Reply by spkane** [reply]
Ah, that makes sense.

I teach a lot of Linux container classes, and while demoing something in my environment, which had the `containerd` image support enabled, I noticed that the `dockerd` daemon was now called `dockerd-c8d`. This discover then sent me on a bit of a side quest after the class ended to figure out what that was all about. I few searches in the `moby` GitHub organization gave me some hints, but nothing super solid, so I thought that  I'd ask.

I also discovered a pretty odd bug with the `containerd` image support and reported an issue earlier today via Google Form that is linked into the Docker Desktop UI for such things. Happy to chat about it via email or in a GH issue.
